pages/linux/csplit.md

```diff
@@ -4,22 +4,26 @@
 > This generates files named "xx00", "xx01", and so on.
 > More information: <https://www.gnu.org/software/coreutils/manual/html_node/csplit-invocation.html>.
 
-- Split a file at lines 5 and 23:
+- Split a file in two parts, starting the second one at line 10:
```

Comment:
This is still sort of ambiguous that are you starting it before or after line 10.